src/sentry/api/serializers/models/group.py

```diff
@@ -460,11 +466,20 @@ def _get_seen_stats(
         perf_issues = [
             group for group in item_list if GroupCategory.PERFORMANCE == group.issue_category
         ]
+        generic_issues = [
+            group
+            for group in item_list
+            if group.issue_category
+            and group.issue_category not in (GroupCategory.ERROR, GroupCategory.PERFORMANCE)
```

Comment:
I feel like I've seen/written code like this a few times now. I wonder if we should have an `is_generic` property or something on the group? Doesn't need to be in this pr, just wondering if it makes sense.

Reply:
Yeah, I think that makes sense to have. I'll add that in a different PR and refactor this as well.